### ISSUE TITLE
2020 January

### ISSUE THREAD

**Issue by amyxie361** [amyxie361/Blog#3]
20200102
Reading:
A Recipe for Training Neural Networks
https://karpathy.github.io/2019/04/25/recipe/
A good training guide for NN from Victor

Interview:
Unity: amilie - 
Amazon: Human Language intelligence lab, Yi Zhang, SJTU, Germany, Synthetic and semantic parsing.
what's an example in a balance between not enough information and quick decision.

**Comment by amyxie361** [comment]
A Survival Guide to a PhD
https://karpathy.github.io/2016/09/07/phd/

http://paperscape.org/
https://www.quora.com/I-got-a-job-offer-from-Google-Facebook-and-Microsoft-and-I-also-got-accepted-into-the-Ph-D-in-Computer-Science-programs-at-MIT-Stanford-and-Berkeley-What-factors-should-I-consider-while-making-a-choice-between-the

**Comment by amyxie361** [comment]
20200105
最近几天在跟victor一起debug 
经验：
git stash / stash pop
branch 代码管理要做好
实验记录： time， path， branch
代码打log应该加上info显示在哪一行

**Comment by amyxie361** [comment]
20200106

CS886 Lecture 1
todo :
word2vec detail -- 实现一下skip-gram
embedding用什么指标eval？除了下游任务
可解释性 http://www.cs.cmu.edu/~ytsvetko/papers/qvec.pdf
人工标注相似性 http://www.cs.technion.ac.il/~gabr/resources/data/wordsim353/
https://wiki.cimec.unitn.it/tiki-index.php?page=CLIC
todo: learn glove
co-occurrence - prob mutual information: word 2 vec as matrix factorization
word embedding 的运算为什么make sense
polysemy一词多义 Arora
https://www.zhihu.com/question/37489735

**Comment by amyxie361** [comment]
20200108
Vic 的bug终于找到了，果然是BERT没初始化成功
lesson：细节永远值得确认

https://github.com/horovod/horovod
Horovod is a distributed deep learning training framework for TensorFlow, Keras, PyTorch, and Apache MXNet. The goal of Horovod is to make distributed deep learning fast and easy to use.

some TODOs in BLEU
Also, many buyers today realize that MT system performance on their specific subject domains and content for different use cases matters much more than how generic systems might perform on news stories.

https://www.cse.iitb.ac.in/~pb/papers/icon07-bleu.pdf
https://www.aclweb.org/anthology/E06-1032.pdf
http://anoopsarkar.github.io/nlp-class/assets/slides/08-evaluation.pdf
https://pbs.twimg.com/media/DQJBTbxV4AApFJE.jpg:large

越南小姐姐
https://huyenchip.com/

**Comment by amyxie361** [comment]
20200109
Annotation 的注意事项：
对比sample得抽一样的才公平

**Comment by amyxie361** [comment]
20200110:
WWW paper gets accepted as short - oral.

**Comment by amyxie361** [comment]
20200113:
interview with 幻方量化
points: main focus: kline, short-time reaction
基本面难以得到历史重现
脑洞：数值也可以做embedding，BERT等成功在于表示学习，所以迁移到其他的也可以做表示学习。

**Comment by amyxie361** [comment]
20200115:
今天主要搞了annotation的分析
统计上面哪些test

**Comment by amyxie361** [comment]
20200116
office hour 小姐姐学习了小黄鸭 https://cyberduck.io/ 貌似同步代码比较科学
jupyter notebook 的优化 https://blog.csdn.net/qq_41662115/article/details/102238367

**Comment by amyxie361** [comment]
20200117
Reversible layers, first introduced in Gomez et al. (2017)
batch normalization（同一个sample normalize）, layer normalization(同一个feature normalize) 功能有啥不同
attention的不同设置？
position embedding 的不同设置

**Comment by amyxie361** [comment]
20200118
Victor's Birthday
Went to watch the musical: The Phantom of the Opera. Great experience!
Then ate Gyubee: Great food and great service.

**Comment by amyxie361** [comment]
20200119
Cooked 麻辣香锅
Went to the Limbo: Shining Ring. And the light show in the Distillation District.

**Comment by amyxie361** [comment]
20200120
课后碰到清华学弟胡
晚饭和Kevin Wu, Gordon Zhang, Allen Wang一起吃的
买了www conference 2020 early bird

20200121
交了改program的表格, 碰到李老师一个master.
跟ali meeting了
午饭和HCI的几个人一起吃
下午把个人网页差不多搞定了

**Comment by amyxie361** [comment]
20200122

今天又没咋好好干活？
沉浸在观察offer中
需要整理一下实验思路

**Comment by amyxie361** [comment]
20200129

面了borealis, 
一面问generative/discriminative model的区别
BERT怎么处理OOV
Baysian 公式
二面coding
三面manager：train/val  曲线，PCA解释，项目细节
感觉不咋滴 对很多问题都没深入想过，有些基础细节也忘了

google meena 细看
https://ai.googleblog.com/

**Comment by amyxie361** [comment]
20200130

上午Lei Li 来演讲 https://lileicc.github.io/
主要做生成的
CGMH https://lileicc.github.io/pubs/miao2019cgmh.pdf
基于HMM，MC采样，句子改写来完成从关键词到句子生成
VTM https://lileicc.github.io/pubs/ye2020variational.pdf
类似于VAE的，据说可以解决VAE生成句子无法控制的问题，是椰蓉的文章
DSS-VAE https://lileicc.github.io/pubs/bao2019generating.pdf 
TODO
还讲了结合视频做解说的 感觉很厉害
也有做一些可解释性的工作
可以follow

**Comment by amyxie361** [comment]
20200131

paper reading
numbers: natural question answering kaggle experience
richard: story generation based on  keyword storyline
me: rev net.

Ji-
early exit bert classification
 
nlp-club:
entity representation
